modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -63,13 +63,17 @@
 /obj/machinery/self_actualization_device/Initialize(mapload)
 	. = ..()
 	update_appearance()
+	create_reagents(300, OPENCONTAINER) // BUBBER EDIT
+	AddComponent(/datum/component/plumbing/simple_demand) // BUBBER EDIT
 
 /obj/machinery/self_actualization_device/close_machine(atom/movable/target, density_to_set = TRUE)
 	..()
 	playsound(src, 'sound/machines/click.ogg', 50)
 	if(!occupant)
 		return FALSE
-	if(!ishuman(occupant))
+	if(!ishuman(occupant) || istype(occupant, /mob/living/carbon/human/species/monkey))// BUBBER EDIT BEGIN
+		say("Warning: Occupant lacks actualizing genes. Please insert synthflesh or select new user. ")
+		playsound(loc, 'sound/machines/chime.ogg', 30, FALSE) // BUBBER EDIT END
```

Comment:
This check is failing in all cases when a mob tries to use the SAD. The SAD is broken.